src/main.cpp

```diff
@@ -108,6 +109,7 @@ class UnicastMessageSenderTask : public AbstractTask<3 * configMINIMAL_STACK_SIZ
             if (!messageSender.processAndSerialize()) {
```

Comment:
Should we check if the socket is still connected to know if it's a connection that changed vs a fail to deserialize?